Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -12280,15 +12280,29 @@ - (void)_insertDynamicImageAnalysisContextMenuItemsIfPossible
             [replacements addObject:[elementAction uiActionForElementInfo:elementInfo]];
         };
 
-        if (foundCopyItem && self.copySubjectResultForImageContextMenu)
-            addAction(_WKElementActionTypeCopyCroppedImage);
+        for (UIMenuElement *child in adjustedChildren.get()) {
+            if (![child isKindOfClass:UIAction.class])
+                continue;
+
+            UIAction *action = (UIAction *)child;
```

Comment:
Please use `dynamic_objc_cast`.

Reply:
clearly I have spent too much time recently in Obj-C land 😂